### ISSUE TITLE
Missing permission

### ISSUE THREAD

**Issue by jan-2017** [lassik/phpbb-ext-telegramnotifications#7]
If I write in a post intended only for admins, the content and information will be sent to the messanger. Everyone can read the telegram message. But since not everyone should see it, it needs an please fix Bye Jan

**Comment by lassik** [comment]
Good call! This didn't occur to me at all. How should we fix this?

1. ACP option to select one group of users. Only messages visible to that group will be sent to Telegram
2. ACP option to select which subforums' messages will be send to Telegram
3. Something else?

Option 1 would be easiest to do. I probably won't have time to do Option 2 myself.

**Comment by jan-2017** [comment]
Yes , Option 1. The permissions are already assigned to the groups in the forum. There should be a feature in the ACP that lets you choose which groups you want to include in the notification. I would offer a selection feature in ACP. If the forum is quite large, for example, I do not want to have every notification from the entire forum to the Messanger, but only what interests me. So a kind of subscriber function.

**Comment by jan-2017** [comment]
small question: how far is the development here.

**Comment by lassik** [comment]
Nobody has done anything yet (as far as I know). Development is slow because I have other priorities, but I could try to do it within 1-2 months.

**Comment by jan-2017** [comment]
thanks for the update

**Comment by lassik** [comment]
No problem, thanks for the continued interest :)
